src/material-experimental/mdc-input/public-api.ts

```diff
@@ -6,6 +6,11 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-export {getMatInputUnsupportedTypeError, MAT_INPUT_VALUE_ACCESSOR} from '@angular/material/input';
 export {MatInput} from './input';
 export {MatInputModule} from './module';
+
+export {
+  getMatInputUnsupportedTypeError,
+  MAT_INPUT_VALUE_ACCESSOR,
+  MatTextareaAutosize,
```

Comment:
This one has a CSS class `.mat-autosize` that needs to become `.mat-mdc-autosize`

Reply:
I've added it to the list of exceptions with a TODO to implement an MDC equivalent.